jep-012-01-raw-string-literals.md

```diff
@@ -0,0 +1,358 @@
+# Raw String Literals
+
+|||
+|---|---
+| **JEP**    | 12-01
+| **Author** | Michael Downling, Maxime Labelle
+| **Status** | accepted
+| **Created**| 19-Nov-2022
+| **Obsoletes**| [JEP-12](./jep-012-raw-string-literals.md)
+
+## Abstract
+
+This JEP proposes the following modifications to JMESPath in order to improve
+the usability of the language and ease the implementation of parsers:
+
+
+* Addition of a **raw string literal** to JMESPath that will allow expressions
+to contain raw strings that are not mutated by JSON escape sequences (e.g.,
+“\\n”, “\\r”, “\\u005C”).
```

Comment:
I expect those to be formatted as JSON strings.

Reply:
"JSON string" seems like a heavily ambiguous concept (it could mean "_JSON text_", or "_JSON text expressing an instance of the string type, [which necessarily begins and ends with `"`]_", or "_an arbitrary sequence of code points represented by JSON text expressing an instance of the string type_"). But regardless, the distinction seems important here, and not currently clear. Perhaps something like this?
```md
* Addition to JMESPath of a **raw string literal** that treats `\` as a
literal character in most cases and in particular does not interpret JSON
escape sequences (and can therefore directly represent strings such as
“\\n”, “\\r”, and “\\u005C” without transformation).
```

Reply:
Indeed I later reverted to use curly quotes for an abstract string and only used double-quotes in this specific paragraph that mentions sample escape séquences.